app/uk/gov/hmrc/plasticpackagingtax/registration/models/genericregistration/GrsJourneyCreationRequest.scala

```diff
@@ -34,4 +34,5 @@ trait GrsJourneyCreationRequest {
   val deskProServiceId: String
   val signOutUrl: String
   val regime: Regime
+  val accessibilityUrl: String
```

Comment:
I guess we just need to run the acceptance tests to verify that GRS is happy with this param and manually check that the correct accessibility page is displayed when we access from one of the GRS pages (we have no tests to prove this).

Reply:
GRS appears to work with when this additional field is past.
The accessibility link does not change. It doesn't look like they've implemented this on their side yet?